app/bundles/DashboardBundle/Controller/DashboardController.php

```diff
@@ -422,7 +424,7 @@ public function importAction()
         $model = $this->getModel('dashboard');
 
         $directories = [
-            'user'   => $this->factory->getSystemPath('dashboard.user'),
+            'user'   => $this->factory->getSystemPath('dashboard.user').$this->factory->getUser()->getId(),
```

Comment:
Please use `$this->user->getId()` instead of `$this->factory->getUser()->getId()` on all places of your PR. We try to avoid using MauticFactory in the new code since it was deprecated.